### ISSUE TITLE
The event object isFirst forever false

### ISSUE THREAD

**Issue by amnotbusy** [hammerjs/hammer.js#1247]
`var myElement = document.getElementById('myElement');

// create a simple instance
// by default, it only adds horizontal recognizers
var mc = new Hammer(myElement);

// let the pan gesture support all directions.
// this will block the vertical scrolling on a touch-device while on the element
mc.get('pan').set({ direction: Hammer.DIRECTION_ALL });

// listen to events...
mc.on("panleft panright panup pandown tap press", function(ev) {
  console.log(ev.isFirst);//forever false
    myElement.textContent = ev.type +" gesture detected.";
});`

**Comment by marcinheniborg** [comment]
Agreed. I'm using pinch event and wanted to set transformOrigin when the first event is triggered. Unfortunately ev.isFirst is always false.
